CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## not released yet
 
+#### Breaking changes
+- Leading slashes in `objectname` are not trimmed anymore when uploading to remote. Before `s5cmd cp /file.txt s3://bucket/` would output `cp /file.txt s3://bucket/file.txt` but now it will output `cp /file.txt s3://bucket//file.txt`. ([#459](https://github.com/peak/s5cmd/pull/459))
```

Comment:
> Before `s5cmd cp /file.txt s3://bucket/` would output `cp /file.txt s3://bucket/file.txt` but now it will output `cp /file.txt s3://bucket//file.txt`

Hmm, this looks wrong to me. `/file.txt` is a file at `/`. Why do we treat it like it has a `/` in its name?

Reply:
> > Before `s5cmd cp /file.txt s3://bucket/` would output `cp /file.txt s3://bucket/file.txt` but now it will output `cp /file.txt s3://bucket//file.txt`
> 
> Hmm, this looks wrong to me. `/file.txt` is a file at `/`. Why do we treat it like it has a `/` in its name?

Sorry, my mistake, it doesn't treat it like that. I was confused by the test files where we use the `putFile` function. However that function is test purposes only and not related to s5cmd's behavior. 

I will remove this part from changelog.

Reply:
Thanks.